src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -4768,6 +4817,46 @@ internal static Collection<string> GetResolvedPathCollection(string filePath, Ex
             return filePaths;
         }
 
+        internal void CreateWindowsPowerShellCompatResources()
+        {
+            lock(WindowsPowerShellCompatSyncObject)
+            {
+                if (WindowsPowerShellCompatRemotingSession == null)
+                {
+                    var commandInfo = new CmdletInfo("New-PSSession", typeof(NewPSSessionCommand));
+                    using(var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace)
+                        .AddCommand(commandInfo)
+                        .AddParameter("UseWindowsPowerShell", true)
+                        .AddParameter("Name", WindowsPowerShellCompatRemotingSessionName))
+                    {
+                        var results = ps.Invoke<PSSession>();
+                        if (results.Count > 0)
+                        {
+                            WindowsPowerShellCompatRemotingSession = results[0];
+                        }
+                    }
+                }
+            }
+        }
+
+        internal void CleanupWindowsPowerShellCompatResources()
+        {
+            lock(WindowsPowerShellCompatSyncObject)
+            {
+                if (WindowsPowerShellCompatRemotingSession != null)
+                {
+                    var commandInfo = new CmdletInfo("Remove-PSSession", typeof(RemovePSSessionCommand));
+                    using(var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace)
+                    .AddCommand(commandInfo)
+                    .AddParameter("Session", WindowsPowerShellCompatRemotingSession))
+                    {
+                        ps.Invoke();
+                        WindowsPowerShellCompatRemotingSession = null;
+                    }
```

Comment:
Maybe C# 8.0 syntax will be more readable:
```c#
                    using(var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace))
                    ps.AddCommand(commandInfo);
                    ps.AddParameter("Session", WindowsPowerShellCompatRemotingSession);
                    ps.Invoke();
                    WindowsPowerShellCompatRemotingSession = null;
 ```

Reply:
The using syntax should be `using var ps = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace);`

Reply:
Changed to 8.0 syntax in all instances.